docs/trace_collection.md

```diff
@@ -183,9 +359,29 @@ Send [traces][1] to Datadog from your Android applications with [Datadog's `dd-s
                     .iterator()
             }
         )
-   val serverSpan = tracer.buildSpan("<SERVER_SPAN_NAME>").asChildOf(extractedContext).start()      
-
-   ```
+        val serverSpan = tracer.buildSpan("<SERVER_SPAN_NAME>").asChildOf(extractedContext).start()      
+    ```
+   {{% /tab %}}
+   {{% tab "Java" %}}
+    ```java
```

Comment:
```suggestion
```java
```

Reply:
@kayayarai I tried to apply the indentation but for some reason when doing the `tab` key is being interpreted as a code block by markdown. Instead I tried to keep them aligned without indentation. Let me know how it looks on your end and if you know a better trick to align them please add a suggestion

Reply:
Understandable. I think it's because we're trying to make it look right on both github and the docs site. It looks alright in the docs now. Just one last comment (added a heading link) in line 375 of trace_collection.md